scripts/start-console.sh

```
@@ -19,7 +19,7 @@ BRIDGE_K8S_MODE_OFF_CLUSTER_THANOS=$(oc -n openshift-config-managed get configma
 BRIDGE_K8S_MODE_OFF_CLUSTER_ALERTMANAGER=$(oc -n openshift-config-managed get configmap monitoring-shared-config -o jsonpath='{.data.alertmanagerPublicURL}')
 BRIDGE_K8S_AUTH_BEARER_TOKEN=$(oc whoami --show-token 2>/dev/null)
 BRIDGE_USER_SETTINGS_LOCATION="localstorage"
-BRIDGE_I18N_NAMESPACES=plugin__monitoring-plugin
+BRIDGE_I18N_NAMESPACES="plugin__${npm_package_consolePlugin_name}"
```

Shellcheck (0.11.0)

[warning] BRIDGE_I18N_NAMESPACES appears unused. Verify use (or export if used externally).

(SC2034)